Update tableau.md (#4380)
fishtown-analytics/docs.getdbt.com: website/docs/docs/use-dbt-semantic-layer/tableau.md

   - Since this is treated as a table, dbt Semantic Layer can't dynamically change what is available. This means we display _all_ available metrics and dimensions even if a particular metric and dimension combination isn't available. 
   
- Certain Table calculations like "Totals" and "Percent Of" may not be accurate when using metrics aggregated in a non-additive way (such as count distinct)

> - In any of our Semantic Layer interfaces (not only Tableau), [any cumulative metric with a time window or grain *must* be looked at with a time dimension](/docs/build/cumulative#limitations)

Reviewer: ```suggestion
- In any of our Semantic Layer interfaces (not only Tableau), you must include a [time dimension](/docs/build/cumulative#limitations) when working with any cumulative metric that has a time window or granularity.
```

Reviewer: slight rephrase
